src/3box.js

```diff
@@ -76,6 +76,10 @@ class Box {
      * @property {Object} spaces            an object containing all open spaces indexed by their name.
      */
     this.spaces = {}
+    /**
+     * @property {Object} spacePubSubMessages   local store of all pinning server pubsub messages seen related to spaces
+     */
```

Comment:
This will become part of the API documentation. I don't think this should be there.

Reply:
right, yeah wanted note for myself, but shouldn't be js doc

Reply:
I guess you can add @private then